### ISSUE TITLE
Documentation help with Service Annotations

### ISSUE THREAD

**Issue by stvdilln** [kubernetes-sigs/aws-alb-ingress-controller#473]
We have a ingress controller with a few rules.  We have a Jenkins server that returns an error on url "/" so we are trying to get the health check to that service to "/robots.txt".  The Documentation says that you can override ingress settings via annotations in the service.  I do not think I have the annotations in the correct place, as they have no effect on the target group.  I've even put very invalid settings in like "health-check-port: goober" and I can't even force an error.  
I've been through the two samples, and don't see a service annotation.  Not sure how this is supposed to work.

Here is how I have tried to annotate the jenkins service:
```
steved@steve-mint ~/projects/github/aws-alb-ingress-controller $ kubectl describe service jenkins
Name:                     jenkins
Namespace:                default
Labels:                   <none>
Annotations:              alb.ingress.kubernetes.io/healthcheck-path=/robots.txt
                          field.cattle.io/publicEndpoints=[{"port":30317,"protocol":"TCP","serviceName":"default:jenkins","allNodes":true}]
Selector:                 app=jenkins
Type:                     NodePort
IP:                       172.20.32.249
Port:                     <unset>  8080/TCP
TargetPort:               8080/TCP
NodePort:                 <unset>  30317/TCP
Endpoints:                10.105.43.93:8080
Session Affinity:         None
External Traffic Policy:  Cluster
Events:                   <none>
```

**Comment by bigkraig** [comment]
@stvdilln which version of the controller are you using? that looks like the correct annotation

**Comment by stvdilln** [comment]
@bigkraig Sheepishly I say 1.0-Alpha 3.  I was following example by pahud, and one of his yaml files was out of date when I deployed.  He's updated his demo instructions and I updated my controller to latest release.  I nuked the alb and let it rebuild, everything is working as expected.
I don't know if the docs could use updating, If I had correct software it would have worked fine.  Probably just close as user-error.  But thanks for the help, I'm up and running!
